gocron.go

```diff
@@ -308,6 +320,12 @@ func (j *Job) mustInterval(i uint64) {
 	}
 }
 
+// BeginNow schedules the next run of the job immediately
+func (j *Job) BeginNow() *Job {
```

Comment:
maybe a better name would be something like "BeginImmediately" or "RunImmediately"? Not sure...

Reply:
Also, correct me if I'm mistaken, but maybe we could achieve the same result by working here with the nextRun variable instead of lastRun?